slides/ml_concepts.md

```diff
@@ -12,8 +12,6 @@ Hi, welcome to the MOOC on machine learning with scikit-learn
 
 class: titlepage
 
-.header[Lesson 1, MOOC Machine learning with scikit-learn]
```

Comment:
I propose that we change this to remove the lesson number, but keep the header, so that if the slides are displayed out of context people know where they come from.

Reply:
For each video, a small generic will be added in post-production to recontextualize the video.

Reply:
By reading the second comment, I realize that it applies to this comment as well. Okay, to leave it on slide 2.